app/models/concerns/galtersufia/generic_file/invenio_resource_type_mappings.rb

```diff
@@ -0,0 +1,180 @@
+        "Architectural Drawings": "Architectural Drawing",
+        "Article":"Journal article;  Newspaper Article; Other Article",
+        "Audio Visual Document": "Video; Sound (Recording)",
+        "Bibliography":	"Bibliography",
+        "Biobibliography": "Biobibliography",
+        "Biography": "Biography; Personal Narrative; Diary",
+        "Blogs": "Social Media Posting",
+        "Case Reports": "Case Report",
+        "Catalogs": "Catalog",
+        "Clinical Trial": "Clinical Trial",
+        "Collected Correspondence": "Collection; Correspondence",
+        "Conference Proceeding": "Conference Proceeding",
+        "Congresses": "Congress",
+        "Dataset": "Dataset",
+        "Dictionary": "Book",
+        "Drawings": "Drawing",
+        "Editorial Article": "Journal article, Editorial",
+        "Electronic Supplementary Materials": "Chart; Table; Dataset; Bibliography; Technical Documentation",
+        "Ephemera": "Ephemera",
+        "Evaluation Studies": "Evaluation Study",
+        "Examination Questions": "Examination Questions",
+        "Exhibitions": "Exhibition",
+        "Fictional Works": "Fictional Work",
+        "Forms": "Form",
+        "Guideline": "Guideline; Study Design",
+        "Handbooks": "Handbook",
+        "Historical Article": "Other Article",
+        "Image": "Pictorial Work, Photograph",
+        "Introductory Journal Article": "Journal Article; Other Article",
+        "Journal Article": "Journal Article",
+        "Journal": "Journal",
+        "Lecture Notes": "Lecture Notes",
+        "Lectures": "Lecture",
+        "Lecture": "Letter; Correspondence",
+        "Manuscript": "Manuscript; Preprint",
+        "Maps": "Map",
+        "Masters Thesis": "Masters Thesis",
+        "Meeting Abstracts": "Meeting Abstract",
+        "News": "News; Newsletter",
+        "Other": "Other; Other Article",
+        "Periodicals": "Magazine; Newsletter",
+        "Photographs": "Photograph",
+        "Pictorial Works": "Pictorial Work",
+        "Poetry": "Poetry",
+        "Portraits": "Portrait",
+        "Postcards": "Postcard",
+        "Poster": "Conference Poster; Other",
+        "Presentation": "Presentation",
+        "Programmed Instruction": "Presentation",
+        "Programs": "Program",
+        "Project": "", #TODO: Check with Sara if this is necessary. According to note from box csv file there are only
+                       # two projects one that is private and one for the collection
+        "Report": "Report",
+        "Research Paper": "Report; Review; Other Article",
+        "Resource Guides": "Resource Guide",
+        "Review": "Review",
+        "Software or Program Code ": "Other; Software; Technical Documentation",
+        "Speech": "Lecture; Presentation; Sound (Recording)",
+        "Statistics": "Statistics",
+        "Study Design": "Study Design",
+        "Tables": "Table",
+        "Technical Report": "Technical Documentation",
+      }.with_indifferent_access
```

Comment:
[HashWithIndifferentAccess](https://api.rubyonrails.org/classes/ActiveSupport/HashWithIndifferentAccess.html) is a fancily named rails class that is really just a hash you can access with symbols OR with strings for keys. Normally hashes only support symbols because they are immutable, but I think for our use case it's fine to do it this way. If nothing else I'm not really sure what the alternative is.

Reply:
Nice! I did not know that about HashWithIndifferentAccess. I like it, and I hope I can remember to use it for a future case. It looks good to me here.